### DISCUSSION TITLE
Worker pods failing almost immediately after creation

### DISCUSSION THREAD

**Discussion by cbc02009** [cloudnative-pg/cloudnative-pg#762] [Q&A] [answered]
I'm running into a strange issue where the postgres worker pods are failing checks almost immediately after being created. I've got logs from the worker and the controller here:

[postgres1.log](https://github.com/cloudnative-pg/cloudnative-pg/files/9602133/postgres1.log)
[postgres-cloudnative-pg.log](https://github.com/cloudnative-pg/cloudnative-pg/files/9602134/postgres-cloudnative-pg.log)

I'm running my kubernetes cluster on Arch linux with kernel 5.19.9-arch1-1. The PVC controller is democratic-csi using NFS shares. The whole setup is using gitops with flux, and the source is available here: https://github.com/cbc02009/k8s-home-ops/tree/0571b107905438b4d623e7fb872cd197d4834803/cluster/manifests/databases/cloudnative-pg

Please let me know if there's any additional information I can provide.

**Comment by cbc02009** [comment]
Okay, this issue is specific to using nfs as a pvc. Maybe I have the wrong settings?

`sharenfs: "rw,no_subtree_check,no_root_squash"`

**Reply by MichaluxPL** [reply]
I'd strongly advice against using NFS for the database purposes . It's not the recommended storage type for PGSQL.

**Reply by cbc02009** [reply]
Yeah, I've moved away from NFS to a local storage provisioner and it seems like the issue is fixed.

**Comment by gbartolini** [comment, marked as answer]
Issue seemed to be related to storage settings - our operator is storage agnostic so I am marking this as resolved.
